src/App.jsx

```diff
@@ -10,6 +11,7 @@ function App() {
       <Header />
       <Switch>
         <Route exact path="/" component={Home} />
+        <Route exact path="/Cards/" component={CardDescription} />
```

Comment:
Why did you add the last "/" in your path ?. 
You don't need to specify exact path for all your routes (just one exact path for your home is enough)